src/WebJobs.Script/CloudBlockBlobHelperService.cs

```diff
@@ -25,18 +28,12 @@ private static async Task<bool> BlobExistsAsync(string url, string environmentVa
                     CloudBlockBlob blob = new CloudBlockBlob(new Uri(url));
```

Comment:
I think everything accessing this is synchronous -- can we use `Exists()` here and make everything sync so we don't do any `GetAwaiter()` stuff?